src/config/config.py

```diff
@@ -0,0 +1,109 @@
+"""Module to handle configuration application configuration parameters."""
+import configparser
+import logging
+from os.path import expanduser, isfile
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+config_file = "config.ini"
+
+
+def find_config_directory() -> str:
+    """Get configuration directory.
+
+    Returns
+    -------
+    str     :
+        Path represented as a string to a directory.
+        Ends with a trailing /
+    """
+    # TODO: Check if windows or linux, use enviroment variables
+    return expanduser("~") + "/.config/nina/"
+
+
+def get_config_file_path() -> str:
+    """Get configuration file path.
+
+    Returns
+    -------
+    str     :
+        Path represented as a string to the configuration file.
+    """
+    return find_config_directory() + config_file
```

Comment:
```suggestion
    return os.path.join(find_config_directory(), config_file)
```

Reply:
maybe it's better that `find_config_directory` returns a Path, and then you just do

```py
return find_config_directory().joinpath(config_file)
```